packages/core/src/lib/normalize.ts

```diff
@@ -1,33 +1,45 @@
-import { type AuditOutputs } from '@code-pushup/models';
+import { type AuditOutputs, Issue } from '@code-pushup/models';
 import { formatGitPath, getGitRoot } from '@code-pushup/utils';
 
+export function normalizeIssue(issue: Issue, gitRoot: string): Issue {
+  // early exit to avoid issue object cloning
```

Comment:
IMHO it is not possible to have sources == null in an issue. 
Should be caught by the schema

Reply:
It is possible, in fact we do it all the time. The `issueSchema` has `source` as optional. All issues from JS Packages plugin don't have a source.

![image](https://github.com/code-pushup/cli/assets/34691111/f9f090d3-7c3f-4c0c-ae9e-01c3a23d4a07)